src/molecules/asides/image-aside/index.js

```diff
@@ -21,7 +21,9 @@ function ImageAside ( props ) {
             icon={ icon }
             wrapperClassName={ styles.icon } />
         :
-          <img src={ icon } className={ styles.icon } />
+          <div className={ styles.icon }>
+            <img src={ icon } />
+          </div>
       }
 
       <aside className={ styles.aside }>
```

Comment:
I think you should update this to use the `TextComponent` -- we don't want this h5 hardwired in, and we'd probably want the TextComponent to handle the typography